itest/lnd_misc_test.go

```diff
@@ -1242,3 +1243,47 @@ func testSignVerifyMessageWithAddr(ht *lntest.HarnessTest) {
 
 	require.False(ht, respValid.Valid, "external signature did validate")
 }
+
+// testInvoiceCltvExpiryFlag ensures that the CltvExpiry option passed to the
+// AddInvoice RPC interface sets `min_final_cltv_expiry_delta` in the invoice
+// and that it fails when a cltv too small or too large is passed.
+func testInvoiceCltvExpiryFlag(ht *lntest.HarnessTest) {
```

Comment:
interested in hearing an opinion from another reviewer too but I personally dont think this warrants an integration test.  This might be more appropriate in a unit test for the  `invoicesrpc.AddInvoice` function

Reply:
The only problem with a functional test is that AFAICT a lot of mocking would be required: we would need a `NodeSigner` and an `InvoiceRegistry` in order to test `invoicesrpc.AddInvoice`. I would be happy to do it if you think that's the right approach

Reply:
To me this might be a case were manual testing alone might be good enough. Especially since no new logic is being tested here. ie, this is not testing something added in this PR. 

If we really want to tests this through an itest, perhaps there is another test we can append...

But again - would like another reviewers opinion here 🙏

Reply:
I've rebased and dropped the tests from this PR